man/man8/zpool.8

```diff
@@ -1449,6 +1458,16 @@ Recovery mode for a non-importable pool. Attempt to return the pool to an import
 .sp
 .ne 2
 .na
+\fB\fB-l\fR\fR
+.ad
+.sp .6
+.RS 4n
+Indicates that the zpool command will request encryption keys for all encrypted datasets it attempts to mount as it is bringing the pool online. This is equivalent to running \fBzfs mount\fR on each encrypted dataset immediately after the pool is imported. If any datasets have a \fBprompt\fR keysource this command will block waiting for the key to be entered. Otherwise, encrypted datasets will be left unavailable until the keys are loaded.
```

Comment:
Thinking about this some more, on the surface it seems like `-l` does approximately nothing, because we already mount filesystems when the pool is imported, and presumably that will load keys for the filesystems we mount.  I think the exception is filesystems with `mountpoint=legacy` or `canmount=off | noauto`.  If that's right, I think it would be worth mentioning here - that `-l` is used to ensure that *all* keys are loaded, even for filesystems that are not mounted because they have `mountpoint=legacy` or `canmount=off | noauto`.